.github/workflows/CI.yml

```diff
@@ -71,3 +71,39 @@ jobs:
       - name: Run sccache for check
         shell: bash
         run: ${SCCACHE_PATH} --start-server
+
+  test_disable_annotations:
+    name: Test disable_annotations version
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+          - windows-latest
+          - macos-latest
+        version:
+          - ""
+          - v0.4.0
```

Comment:
why this version?

Reply:
this is copy from existing job, maybe we should keep a single job on ubuntu only?